### PR TITLE
Substitute akismet package and reconfigure spam filtering

### PR DESCRIPTION
**Pull request by VirginiaDooley**
In this PR, I've swapped the `python-akismet` spam filtering package for `akismet` with the (so far unproved) theory that a package that is maintained more regularly/recently would help filter out increasingly sophisticated spam submissions. 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208159469948700

### CODE REVIEW

File: wcivf/settings/base.py [hunk 1]
```diff
@@ -279,9 +279,9 @@ def get_ec2_ip():
 # DC Base Theme settings
 SITE_TITLE = "Who Can I Vote For?"
 
-AKISMET_API_KEY = os.environ.get("AKISMET_API_KEY")
+PYTHON_AKISMET_API_KEY = os.environ.get("PYTHON_AKISMET_API_KEY")
```
Comment (symroe) [inline wcivf/settings/base.py:282]: If you want to change this environment viable, you'll need to change it in the parameter store. 

However, I don't really see the need to change the var name here. The content is the same, and for me `AKISMET_API_KEY` is better than `PYTHON_AKISMET_API_KEY`...in the context of a python project the prefix feels redundant to me.

Comment (VirginiaDooley) [reply wcivf/settings/base.py:282]: I changed this on the advice on the package installation instructions https://akismet.readthedocs.io/en/latest/sync_client.html but happy now that I'm setting the config variable, I can just revert and do: 

`config = akismet.Config(
            key=settings.PYTHON_AKISMET_API_KEY,
            url=settings.PYTHON_AKISMET_BLOG_URL,
        )`

File: wcivf/settings/base.py [hunk 2]
```diff
@@ -281,7 +281,7 @@ def get_ec2_ip():
 
 AKISMET_API_KEY = os.environ.get("AKISMET_API_KEY")
 
-AKISMET_BLOG_URL = CANONICAL_URL
+PYTHON_AKISMET_BLOG_URL = CANONICAL_URL
```
Comment (symroe) [inline wcivf/settings/base.py:284]: You're still using `AKISMET_BLOG_URL`, so can you revert this line too?

Comment (VirginiaDooley) [reply wcivf/settings/base.py]: That's done now, thanks for catching.

**Review by symroe — APPROVED** [review]

**Pull request merged**